template/src/router.js

```diff
@@ -0,0 +1,17 @@
+import Vue from 'vue'{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+import VueRouter from 'vue-router'{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
+import { Hello } from './components'{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
```

Comment:
@chrisvfritz This import is not working. Can you check it once?

I had to do the following to get it working:

```js
import Hello from './components/Hello';
```